sdk/spring-sdk/src/main/java/kalix/springsdk/view/ReflectiveViewProvider.java

```diff
@@ -62,7 +63,7 @@ private ReflectiveViewProvider(
       Function<ViewCreationContext, V> factory,
       ViewOptions options) {
     this.factory = factory;
-    this.options = options;
+    this.options = options.withForwardHeaders(ForwardHeadersExtractor.extractFrom(cls));
```

Comment:
I added forward headers support for views, but I'm not sure if there is an actual use case for that. GET - methods should be without implementation and subscription methods shouldn't be called from the outside.

Reply:
Yep, I agree. I don't see I reason to have support for forward headers in views, but maybe @johanandren has some extra insight about it? Maybe we added it for some future use cases?

Reply:
Will not be useful until/if we add view result transformations in user code. No strong opinion whether adding is worth it or not.

Reply:
But even if we have result transformation done on the UF side, does it makes sense to have forward headers? The proxy will be calling that transformer method. We won't have an external client that could potentially send some headers that users could be interested.

Reply:
Only for the query method(s). When/if we add query request/result transform, headers the user logic could use the request headers to transform the request parameters, result rows or even completely deny a request based on some headers.

Reply:
I don't see how it would be possible to have query methods that also run transformations. Query methods have query input as parameter. How would we pass the row content for transformation? 

Anyway, we can discuss that in detail the if/when we decide about it. 

For now, I would simply not support forward headers in views because we simply can't support it.

Reply:
Removed